### PR TITLE
[FIX] project: use website lang for portal sharing view

### PR DESCRIPTION
**Pull request by fdamhaut**
Step to reproduce:
- Have a least 2 lang with portal lang set to langA
- Log in as portal user
- In portal change lang to langB
- Go to portal sharing view

Current behaviour:
- Iframe does not get info from the request and use the user's
 lang

Behaviour after PR:
- If there is a lang set on the website (in url) we use this one
instead.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by xavierbol — APPROVED** [review]
LGTM, thanks for the fix, I let @IT-Ideas doing the final review. 🙂

File: addons/project/views/project_sharing_templates.xml [hunk 1]
```diff
@@ -9,7 +9,8 @@
     </template>
 
     <template id="project_sharing" name="Project Sharing View">
-        <iframe width="100%" height="100%" frameborder="0" t-attf-src="/my/project/{{ project_id }}/project_sharing"/>
+        <!--    We need to forward the request lang to ensure that the lang set on the portal match the lang delivered -->
+        <iframe width="100%" height="100%" frameborder="0" t-attf-src="/{{ request.context['lang'] }}/my/project/{{ str(project_id) }}/project_sharing"/>
```
Comment (IT-Ideas) [inline addons/project/views/project_sharing_templates.xml:13]: Wouldn't it be safer to check that 'lang' is in the context? Or is it always present?

**Review by IT-Ideas — COMMENTED** [review]
Thanks 🙂 
Just a question.

File: addons/project/views/project_sharing_templates.xml (hunk 1 above)
Comment (fdamhaut) [reply addons/project/views/project_sharing_templates.xml:13]: > Wouldn't it be safer to check that 'lang' is in the context? Or is it always present?

When there is no lang in the URL, it default to en_US.
You can try by just testing this with no additional language set :D

Comment (IT-Ideas) [reply addons/project/views/project_sharing_templates.xml:13]: Also if the key could be an empty string that would become `//my/project/{{ str(project_id) }}/project_sharing` which I am unsure it is a valid url.
But as you told me that could not happen, I am fine.

**Review by IT-Ideas — APPROVED** [review]
Thanks 🙂 
@robodoo r+

**Comment by IT-Ideas** [comment]
@robodoo r+

**Pull request closed without merging**